src/components/search/SuggestionProductCard/suggestion-product-card.scss

```diff
@@ -3,6 +3,10 @@
 .suggestion-product-card {
```

Comment:
```suggestion
[data-suggestion-product-card] {
```

Reply:
Addressed it [here](https://github.com/vtex-sites/gatsby.store/pull/7/commits/3802754f668a0b7f88cbcdd6e539177ccfefc92c).